src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -171,6 +171,22 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
     }
     
     /**
+     * Generates the feedback session closed emails for the given {@code session}.
+     */
+    public List<EmailWrapper> generateFeedbackSessionClosedEmails(FeedbackSessionAttributes session) {
+
+        boolean isEmailNeeded = fsLogic.isFeedbackSessionForStudentsToAnswer(session);
```

Comment:
@wkurniawan07 we are not sending closing emails also if the session is for only instructors. I thought that was intentional. Should I check if the session is not private send to all instructors?